tests/contrib/lcsh/test_converter.py

```diff
@@ -48,16 +49,65 @@ def test_deprecated():
     filepath = Path(__file__).parent / "data" / "fake_lcsh.skosrdf.jsonld"
     topics = list(read_jsonl(filepath))
 
-    dicts_of_replacements = [d for d in deprecated_to_replacements(topics)]
+    dicts_of_replacements = [d for d in raw_to_deprecated(topics)]
 
     expected = [
+        # No clear new_id/new_subject
         {
             "id": "https://id.loc.gov/authorities/subjects/sh00000273",
             "time": "2021-07-20T08:25:20",
             "subject": "Child concentration camp inmates",
-            "new_id": "TODO:Expert:Fill me",
-            "new_subject": "TODO:Expert:Fill me",
+            "new_id": "",
+            "new_subject": "",
             "notes": "This authority record has been deleted because the heading is covered by the subject headings {Child internment camp inmates} (DLC)sh2021004026 and {Child Nazi concentration camp inmates} (DLC)sh2021004027"  # noqa
```

Comment:
With Gretchen, we established to not associate any replacement id/subject in this case and let the curator manually select which should be more appropriate.